### PR TITLE
Add ABOUT page.

### PR DESCRIPTION
**Pull request by ramiroluz**

### CODE REVIEW

**Comment by kytrinyx** [comment]
We have two PRs right now for the ABOUT.md: See also #275

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+##  About Python
```
Comment (kytrinyx) [inline docs/ABOUT.md]: This gets incorporated on a page that already has a header, so this first line can go away.

**Comment by ramiroluz** [comment]
Chose one of the pull requests, I do the merge of the other.

**Comment by Dog** [comment]
I went ahead and merged #275 for you.

**Comment by kytrinyx** [comment]
@ramiroluz Would you please rebase this against the upstream master?

**Comment by behrtam** [comment]
I will close this abandoned pr for clarity. If you want to continue your work on this, we can reopen it.

**Comment by ramiroluz** [comment]
I am very sorry, I missed the message from @kytrinyx. Have just rebased my fork with upstream/master. Should I make a new pull request?

**Comment by behrtam** [comment]
You cold continue here. Opened it again.

**Comment by ramiroluz** [comment]
I deleted my fork, created a new fork. Applied the changes to the About page and created a fresh new pull request.

https://github.com/exercism/xpython/pull/390

**Pull request closed without merging**